tests/test_physics_unit_tests.py

```diff
@@ -0,0 +1,51 @@
+import itertools
+import multiprocessing
+import runpy
+import sys
+from os import path as osp
+
+import pytest
+
+
+def run_main(*args):
+    # patch sys.args
+    sys.argv = list(args)
+    target = args[0]
+    # run_path has one difference with invoking Python from command-line:
+    # if the target is a file (rather than a directory), it does not add its
+    # parent directory to sys.path. Thus, importing other modules from the
+    # same directory is broken unless sys.path is patched here.
+    if osp.isfile(target):
+        sys.path.insert(0, osp.dirname(target))
+    runpy.run_path(target, run_name="__main__")
+
+
+def powerset(iterable):
+    s = list(iterable)
+    return itertools.chain.from_iterable(
+        itertools.combinations(s, r) for r in range(len(s) + 1)
+    )
```

Comment:
This is not required here and should be removed, you're not using it as you formulate your test.